src/libs/utils.js

```diff
@@ -11,6 +11,10 @@ export const formatDate = (dateval) => {
   return datestr;
 };
 
+export const isObjectEmpty = (obj) => {
```

Comment:
Could not use lodash's  _.isEmpty() ?

Reply:
Thank you - good idea. I am not that familiar with lodash ... need to spend some time with it.

Reply:
me too. I just saw it is on the project and have that function.